app/services/trainees/create_from_hesa.rb

```
@@ -15,13 +15,14 @@ def initialize(student_node:)
     def call
       Audited.audit_class.as_user(USERNAME) do
         trainee.assign_attributes(mapped_attributes)
-        trainee.save!
 
-        create_degrees!
-        add_multiple_disability_text!
-
-        trainee
+        if trainee.save
```

We should probably `save!` here as this will fail silently otherwise I think.